README.md

```diff
@@ -66,139 +69,222 @@ or
   :load-path "/path/to/lib")
 ```
 
-## 📇 Commands
+## 🖥 Usage
+
+### 📇 Commands
 
-| Commands                   | Description                                                                 |
+The following are the functions provided by `ts-fold-mode`
```

Comment:
Maybe add a newline after this line so it's not right next to the table.

Reply:
Oh, this was the reason for the table not formatting properly! Thank you for pointing this out!